controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -73,10 +73,11 @@ class ControllerManager : public rclcpp::Node
     const std::string & controller_name,
     const std::string & controller_type);
 
+  /// load_controller loads a controller by name, the type must be defined in the parameter server.
   /**
-   * @brief load_controller loads a controller by name, the type must be defined in the parameter server
-   * @param controller_name as a string.
-   * @see Documentation in controller_manager_msgs/LoadController.srv
+   * \param[in] controller_name as a string.
+   * \return controller_interface::ControllerInterfaceSharedPtr controller
```

Comment:
I reiterating the type name here extremely redundant and prone to breaking. Doxygen knows how to generate that part,we should only supply text that explains what the code may not. 
@Karsten1987 is there a less verbose version you'd recommend?

Reply:
So, should I remove the <type> from the style?

Just for notice, It is already exists C++ typenames in docstrings in some files:
https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/include/hardware_interface/actuator_interface.hpp#L95
https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/src/component_parser.cpp#L111

Reply:
Yeah I am aware we already have precedent for it but precedent doesn't mean it's right ;)  Let's wait for Karsten's take on this before you spend time

Reply:
I agree, I don't necessarily think we have to mention the data type here.

Reply:
Ok, I removed the type from the style.